hexa/visualizations/models.py

```diff
@@ -65,7 +65,7 @@ class ExternalDashboard(IndexableMixin, models.Model):
     picture = models.FileField(upload_to="external_dashboard")
     credentials = EncryptedTextField(null=True, blank=True)
 
-    indexes = GenericRelation("dashboards.Index")
+    indexes = GenericRelation("visualizations.Index")
```

Comment:
(Not related to this line but can't comment above)

Queryset should be `ExternalDashboardQuerySet`

Reply:
Also `ExternalDashboardPermission.__str__()`: should be `self.external_dashboard`